src/net.cpp

```diff
@@ -692,23 +704,14 @@ CNetMessage V1TransportDeserializer::GetMessage(const CMessageHeader::MessageSta
     // store state about valid header, netmagic and checksum
```

Comment:
This should be changed to read something like:
```suggestion
    // store state about valid header and netmagic
```